config/crd/experimental/kustomization.yaml

```diff
@@ -8,4 +8,4 @@ resources:
 - gateway.networking.k8s.io_udproutes.yaml
 - gateway.networking.k8s.io_grpcroutes.yaml
 - gateway.networking.k8s.io_backendtlspolicies.yaml
-
+- gateway.networking.k8s.io_backendlbpolicies.yaml
```

Comment:
Would be best to maintain trailing newline here to keep linters happy.

Reply:
Oh, I didn't notice that

Reply:
Do you mean whole empty line, as of two newlines? Because there were two before my commit and now there is one.

Reply:
Oh interesting, looks like I misread the diff, your change is correct.